pkgs/development/python-modules/ansible/default.nix

```
@@ -14,6 +14,7 @@
 , dopy
 , windowsSupport ? false
 , pywinrm
+, hcloud
```

since this isn't part of the install_requires on setup.py, do you mind adding a comment?
```suggestion
, hcloud # hcloud plugin
```